lms/templates/footer-edx-new.html

```diff
@@ -31,12 +31,20 @@ <h2 class="sr footer-about-title">${_("About edX")}</h2>
       </div>
 
       <div class="footer-about-copyright">
-        ## Translators: '&copy;' is an HTML character code for the copyright symbol. Please don't remove or change it.
-        <p>${_("&copy; {copyright_year} {platform_name}, some rights reserved").format(
+	<p>&copy; ${settings.COPYRIGHT_YEAR} ${settings.PLATFORM_NAME}.</p>
+        <p>${"&copy; {copyright_year} {platform_name}".format(
```

Comment:
This would be better as
```mako
<p>&copy; ${settings.COPYRIGHT_YEAR} ${settings.PLATFORM_NAME}</p>
```
That way, you don't have to do the string formatting.